lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -1448,7 +1448,7 @@ def generate_students_certificates(
             course=course
         )
 
-        if status in [CertificateStatuses.generating, CertificateStatuses.downloadable]:
+        if CertificateStatuses.is_passing_status(status):
```

Comment:
I answered the same question from Renzo up here https://github.com/edx/edx-platform/pull/11341#discussion_r51930027 - basically, that module uses certificates models all over the place (instead of the api), and I didn't want to add a larger refactoring of that module into this PR.